contracts/SimpleMultiSignature.sol

```diff
@@ -70,6 +73,9 @@ contract SimpleMultiSignature is EIP712 {
     uint256 nonce,
     bytes memory signatures
   ) external returns (bool success) {
+    // Check that the contract is not paused
+    require(!_paused, "SimpleMultiSignature: Contract is paused");
```

Comment:
You forgot to define _paused in the contract storage

Reply:
ah oui j'ai pas fais attention